### ISSUE TITLE
Improve detect openscad executable using `which`

### ISSUE THREAD

**Issue by jeff-dh** [jreiberkyle/jupyterscad#6]
On my system openscad is installed in `/home/xxx/.local/bin`. Your detect executable function does not find it.

A call to `which` would provide a valid path to the executable:

```python
>>> from shutil import which
>>> which("openscad")
'/home/xxx/.local/bin/openscad'
>>> 
```

**Comment by jreiberkyle** [comment]
Out of curiosity, what os/install path are you using?

**Comment by jeff-dh** [comment]
Linux, /home/xxx/.local/bin

**Comment by jreiberkyle** [comment]
closed by #18
